dev/openchpl_preload.sql

```diff
@@ -1892,22 +1892,22 @@ INSERT INTO openchpl.qms_standard(name, last_modified_user) VALUES
 ('IEC 62304', -1);
 
 INSERT INTO openchpl.activity_concept (activity_concept_id, concept, last_modified_user) VALUES
```

Comment:
Don't think this change will work because the INSERT statement still lists the activity_concept_id column. Make sure to run reset-test-only.sh after changing this to confirm no errors.